603-spring-web-smallrye-openapi/src/test/java/com/baeldung/springbootbootstrap/HomePageTest.java

```diff
@@ -0,0 +1,23 @@
+package com.baeldung.springbootbootstrap;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+
+@QuarkusTest
+public class HomePageTest {
+    private static final String APP_NAME = "Bootstrap Spring Boot";
+
+    @Test
+    public void testHomePage() {
```

Comment:
As far I understood, you're covering the qute extension here. What about renaming the test method to shouldQuteReplaceWelcomePharse or something similar?